gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.8.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.9-bin.zip
```

Comment:
this won't actually update the gradle wrapper jar, you need to run `./gradlew wrapper --gradle-version 4.9 --distribution-type bin`

only for version 2.14 can you just change the gradle-wrapper.properties

Reply:
mmh ok, I'll work on it tomorrow, I've tried updating it now, but it seems like it can't download forgegradle anymore so I'll need to fix that before pushing.
although I'm wondering why should we update the wrapper, it worked just fine before in my env

Reply:
oof ok

I just want gradle 4.9 because it's the one most 1.12 mods use (and i'm pretty low on storage so I don't want another version of gradle in my caches folder)
tl;dr convenience